Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -221,11 +221,83 @@ script:
       name: member
       required: true
       secret: false
-    deprecated: false
+    deprecated: true
     description: Adds a member (user) to a private channel.
     execution: false
     name: add-user-to-channel
-  dockerimage: demisto/teams:1.0.0.1701
+  - arguments:
+    - default: false
+      description: The user to call.
+      isArray: false
+      name: username
+      required: true
+      secret: false
+    - default: false
+      description: The tenant ID for the Teams.
+      isArray: false
+      name: tenant_id
+      required: true
+      secret: false
+    - default: false
+      description: The callback URI for the call for the bot who will recive a callback
```

Comment:
it is still not clear what should be passed here..

Reply:
I'll add it to the integration description - it might be too much for an argument description

Reply:
dont think integration description is the place for that..